wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/BitbucketServerPersonalAccessTokenFetcher.java

```diff
@@ -179,21 +178,8 @@ public Optional<Pair<Boolean, String>> isValid(PersonalAccessTokenParams params)
       }
     }
     try {
-      // Token is added manually by a user without token id. Validate only by requesting user info.
-      if (isNullOrEmpty(params.getScmTokenId())) {
-        BitbucketUser user = bitbucketServerApiClient.getUser(params.getToken());
-        return Optional.of(Pair.of(Boolean.TRUE, user.getName()));
-      }
-      // Token is added by OAuth. Token id is available.
-      BitbucketPersonalAccessToken bitbucketPersonalAccessToken =
-          bitbucketServerApiClient.getPersonalAccessToken(
-              params.getScmTokenId(), params.getToken());
-      return Optional.of(
-          Pair.of(
-              DEFAULT_TOKEN_SCOPE.equals(bitbucketPersonalAccessToken.getPermissions())
-                  ? Boolean.TRUE
-                  : Boolean.FALSE,
-              bitbucketPersonalAccessToken.getUser().getName()));
```

Comment:
@vinokurig could BitBucket OAuth be affected by this change? why scope validation was in place before?

Reply:
> could BitBucket OAuth be affected by this change?

I do not think so because the commit just simplifies the check, it does not add new requests or stuff like that.
> why scope validation was in place before?

I believe this was copied from the GitHub token fetcher: https://github.com/eclipse-che/che-server/blob/88b965fbca111881c5db80e577d407b0190738f7/wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubPersonalAccessTokenFetcher.java#L238-L243